### PR TITLE
Function to find similar nodegroups

### PR DESCRIPTION
**Pull request by MaciekPytel**
Ref: #47

### CODE REVIEW

File: cluster-autoscaler/utils/nodegroupset/node_group_set.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			errors.InternalError,
+			"failed to find template node for node group %s",
+			nodeGroupId)
+	}
+	for _, ng := range cloudProvider.NodeGroups() {
+		ngId := ng.Id()
+		if ngId == nodeGroupId {
+			continue
+		}
+		ngNodeInfo, ok := nodeInfosForGroups[ngId]
```
Comment (mwielgus) [inline cluster-autoscaler/utils/nodegroupset/node_group_set.go]: s/ok/found

File: cluster-autoscaler/utils/nodegroupset/node_group_set.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		if !ok {
+			return []cloudprovider.NodeGroup{}, errors.NewAutoscalerError(
```
Comment (mwielgus) [inline cluster-autoscaler/utils/nodegroupset/node_group_set.go]: Not sure if this is an error. We may not be able to find some nodeInfos for some node group and overall this should not be a critical problem. The worst thing that can happen is that we will skip some nodegreoups. Please change it to glog.Warning + continue

Comment (MaciekPytel) [inline cluster-autoscaler/utils/nodegroupset/node_group_set.go]: done

File: cluster-autoscaler/utils/nodegroupset/node_group_set.go (hunk 1 above)
Comment (MaciekPytel) [inline cluster-autoscaler/utils/nodegroupset/node_group_set.go]: done

**Comment by mwielgus** [comment]
lgtm

**Pull request merged**